mantidproject/mantid
# Create python algorithm SelectNexusFilesByMetadata

This PR adds a new python algorithm for filtering of multiple nexus files by metadata without loading.

**To test:**


1. Collect some set of files.
2. Pick some nexus metadata entries, and build some criteria from them.
3. Run the algorithm following the example in docs.
4. Check in the returned files, that the criterion is indeed satisfactory.
5. Try more edge cases you can think of.

Fixes #17272 .



---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
